frontend/src/scenes/cohorts/cohortLogic.ts

```diff
@@ -31,7 +32,17 @@ function determineMatchType(group: Partial<CohortGroupType>): MatchType {
 function processCohortOnSet(cohort: CohortType): CohortType {
     if (cohort.groups) {
         cohort.groups = cohort.groups.map((group) => addLocalCohortGroupId(group))
+        cohort.groups = cohort.groups.map((group) => {
```

Comment:
nit: can we do this in one loop instead of two?

Reply:
yea I just didn't want to touch existing code too much, will consider for the next PR